### DISCUSSION TITLE
Do we need annotations to be sortable?

### DISCUSSION THREAD

**Discussion by llemeurfr** [readium/annotations#13] [General]
Some locators are easily sortable (EPUB CFI), other are not (Text quote selectors). 

Sorting annotations is useful if an app needs to export annotations sorted in "reading order". This is explicitly the case if the app exports a human-readable structure (usually html ready for print).

**Comment by HadrienGardeur** [comment]
I think sortability is good to have but not necessarily a blocker.

It's important to keep in mind that we'll also deal with multiple selectors most of the time. While a text quote selector isn't sortable, an accompanying position could be.

**Reply by llemeurfr** [reply]
I agree with this take. Serializing progression selectors (and handling them in reading apps for sorting outputs) is an efficient solution to the non-sortability of some other selectors, and should be recommended.

**Comment by llemeurfr** [comment]
I added a section named "Using multiple selectors" in the "Best Practices for Reading Systems" section. It contains:

> It is recommended that Reading Systems export Progression Selectors, as they can be used for sorting annotations when no other sortable selector is present.
